src/libraries/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -32,7 +32,6 @@ public sealed partial class Cookie
         public bool Expired { get { throw null; } set { } }
         public System.DateTime Expires { get { throw null; } set { } }
         public bool HttpOnly { get { throw null; } set { } }
-        [System.Diagnostics.CodeAnalysis.AllowNullAttribute]
```

Comment:
This is a breaking change against .NET 5.0, however the implementation indeed doesn't allow null in this case:

https://github.com/dotnet/runtime/blob/632a56b089b03e107ea20b2a9276e6235c4e01ac/src/libraries/System.Net.Primitives/src/System/Net/Cookie.cs#L230-L233

Reply:
IIRC we are still permitting corrections to nullable annotations?

Reply:
I think we are. cc: @stephentoub @jeffhandley

Reply:
Yup.